app/views/users/_profile_header.html.erb

```diff
@@ -7,23 +7,34 @@
   .user-profile-follow-button {
     border: 1px solid <%= user_colors(@user)[:text] %> !important;
   }
-
   <% end %>
 </style>
-<div class="user-profile-header" style="<%= user_colors_style(@user) %>" itemscope itemtype="http://schema.org/Person" itemprop="mainEntity">
-  <meta itemprop="url" content="https://dev.to/<%= @user.username %>">
+
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Organization",
+    "mainEntityOfPage": "True",
```

Comment:
according to https://schema.org/mainEntityOfPage this should be a property, correct?